lib/ruby_lsp/requests/document_highlight.rb

```diff
@@ -47,19 +47,29 @@ def initialize(target, parent, emitter, message_queue)
 
         highlight_target =
           case target
-          when *DIRECT_HIGHLIGHTS
+          when YARP::GlobalVariableReadNode, YARP::GlobalVariableAndWriteNode, YARP::GlobalVariableOperatorWriteNode,
```

Comment:
If wonder if there's some way to structure this list would be less sensitive to changes in YARP (e.g. renamed, added, or deleted node types). Maybe @kddnewton has thoughts?

Reply:
I see what you mean, but hopefully the AST should be pretty stable going forward. Most of the churn has been while we've been getting it up and running.